reference/cd-github.md

```diff
@@ -0,0 +1,92 @@
+---
+title: GitHub Workflow (Preview)
+---
+
+<style>
+    .img-bordered {
+        border: 2px solid rgba(0,0,0,0.15);
+    }
+    .move-right {
+        margin-left: 32px;
+    }
+    .move-right-small {
+        margin-left: 8px;
+    }
+</style>
+
+Pulumi's GitHub workflow integration is a feature which shows the results of Pulumi stack
+updates on GitHub. Showing you any potential infrastructure changes on Pull Requests,
+as well as recording stack changes made with each push.
+
+> GitHub workflow integration is currently in preview, and we will be expanding the feature set
+> over the comming weeks and months. If you have any feedback, suggestions, or bug reports, please
+> let us know in the [Pulumi Community Slack](https://slack.pulumi.io/).
+
+
+**Pull Request Comment**
+
+![Comment on Pull Request](/images/github-app/pr-comment.png){:class="img-bordered"}{:class="move-right"}
```

Comment:
I'd add some text. For instance,

The Pulumi GitHub App will automatically add comments to PRs that change resources with information about previews and updates. This includes a summary of how many resources were created, updated, and/or deleted. This integration allows you to more easily see changes caused by your Pulumi program without needing to leave GitHub's Pull Requests view. Rich details are available by clicking the link to Pulumi.com.

Reply:
Sounds good, done.